paywall/src/utils/validators.js

```diff
@@ -119,6 +120,16 @@ export const isValidPaywallConfig = config => {
     return false
   }
 
+  if (
+    !(
+      typeof config.persistentCheckout === 'boolean' ||
```

Comment:
could we name it `showCheckoutWhenLocked` instead so it describes what behavior it will elicit?

Reply:
hum. I have no idea what `showCheckoutWhenLocked` means? 
It is about not letting the user close the modal when set to `true`.

Reply:
ok. so what happens when a key is purchased? can the user close the modal then?

Reply:
Exactly like in the "paywall" type!

Reply:
`persistentCheckout` implies that it is always there. I'm suggesting a name that captures this behavior: The checkout can't be closed when the content is locked. How about `whenLockedCheckoutPersists`